zone/mob.cpp

```diff
@@ -1112,7 +1114,7 @@ void Mob::FillSpawnStruct(NewSpawn_Struct* ns, Mob* ForWho)
 	ns->spawn.spawnId	= GetID();
 	ns->spawn.curHp	= static_cast<uint8>(GetHPRatio());
 	ns->spawn.max_hp	= 100;		//this field needs a better name
-	ns->spawn.race		= race;
+	ns->spawn.race		= (use_model) ? use_model : race;
```

Comment:
First glance I'm not really use what `use_model` means, if the purpose of this is to be able to provide a "Temporary race" why not just have it be `temporary_race` which makes it far more obvious to new eyes

Reply:
So if you have a gnoll in highpass - the base db uses the new race for (Gnoll New).  It does this not because the mob isn't a gnoll, but because they want the new graphics.  The use_model field lets you change race back to the old gnoll race (making gnollbase weapons work, as well as any other stuff in the db or code that applies to gnolls) and put the model to use in this new field.

Basically it decouples model from race for any mob where it is filled in.